### ISSUE TITLE
Add filter for sent surveys

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/pictureapp#1010]
In the sent surveys fragment, we have to add a dropdown at the top with the following options:
This week
This month
Last week
Last month
Last 6 days
Last 6 weeks
Last 6 months

The default option will be "This week". That dropdown selection will cause the unsent list to show only surveys in that period. 

Right to that dropdown there will be a text saying the number of cases that match the filter (i.e. 32 cases).

To sum up:
- [x] Add dropdown at the top of sent surveys with those options (default to "this week")
- [x] Filter sent surveys by that selection 
- [x] Show the number of cases right of the dropdown
